src/main/java/org/privacyidea/PIResponse.java

```diff
@@ -16,186 +16,52 @@
-            this.status = getBoolean(result, STATUS);
-            this.value = getBoolean(result, VALUE);
-
-            JsonElement errElem = result.get(ERROR);
-            if (errElem != null && !errElem.isJsonNull()) {
-                JsonObject errObj = result.getAsJsonObject(ERROR);
-                this.error = new Error();
-                this.error.code = getInt(errObj, CODE);
-                this.error.message = getString(errObj, MESSAGE);
-            }
-        }
-
-        JsonElement detailElem = obj.get(DETAIL);
-        if (detailElem != null && !detailElem.isJsonNull()) {
-            JsonObject detail = obj.getAsJsonObject(DETAIL);
-            this.message = getString(detail, MESSAGE);
-            this.serial = getString(detail, SERIAL);
-            this.transaction_id = getString(detail, TRANSACTION_ID);
-            this.type = getString(detail, TYPE);
-            this.otplen = getInt(detail, OTPLEN);
-
-            JsonArray arrMessages = detail.getAsJsonArray(MESSAGES);
-            if (arrMessages != null) {
-                arrMessages.forEach(val -> {
-                    if (val != null) {
-                        this.messages.add(val.getAsString());
-                    }
-                });
-            }
-
-            JsonArray arrChallenges = detail.getAsJsonArray(MULTI_CHALLENGE);
-            if (arrChallenges != null) {
-                for (int i = 0; i < arrChallenges.size(); i++) {
-                    JsonObject challenge = arrChallenges.get(i).getAsJsonObject();
-                    if (TOKEN_TYPE_WEBAUTHN.equals(getString(challenge, TYPE))) {
-                        String webAuthnSignRequest = "";
-                        JsonElement attrElem = challenge.get(ATTRIBUTES);
-                        if (attrElem != null && !attrElem.isJsonNull()) {
-                            JsonElement webauthnElem = attrElem.getAsJsonObject().get(WEBAUTHN_SIGN_REQUEST);
-                            if (webauthnElem != null && !webauthnElem.isJsonNull()) {
-                                webAuthnSignRequest = webauthnElem.toString();
-                            }
-                        }
-                        multichallenge.add(new WebAuthn(
-                                getString(challenge, SERIAL),
-                                getString(challenge, MESSAGE),
-                                getString(challenge, TRANSACTION_ID),
-                                webAuthnSignRequest
-                        ));
-                    } else {
-                        multichallenge.add(new Challenge(
-                                getString(challenge, SERIAL),
-                                getString(challenge, MESSAGE),
-                                getString(challenge, TRANSACTION_ID),
-                                getString(challenge, TYPE)
-                        ));
-                    }
-                }
-            }
-        }
-    }
+    public String message;
```

Comment:
Again, the response could contain additional information like the user attributes in the ``detail``, if it is used for SAML server, which use privacyIDEA as primary auth source.